pkg/reconciler/apis/identitycache/api_export_identity_controller.go

```diff
@@ -90,17 +90,16 @@ func NewApiExportIdentityProviderController(
 
 	configMapInformer.Informer().AddEventHandler(cache.FilteringResourceEventHandler{
 		FilterFunc: func(obj interface{}) bool {
-			key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
```

Comment:
@ncdc @sttts do we really want `obj interface -> clusterName logicalcluster.Name` transformation to be implemented like this? Why not a `logicalcluster.FromAny(obj interface) (logicalcluster.Name, error)`?